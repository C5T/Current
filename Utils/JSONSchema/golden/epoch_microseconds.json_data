{"us":430124400000000,"comment":"date -d 'Aug 19 1983' +'%s000000'"}
{"us":745743600000000,"comment":"date -d 'Aug 19 1993' +'%s000000'"}
{"us":1376895600000000,"comment":"date -d 'Aug 19 2013' +'%s000000'"}
{"comment":"expecting std::chrono::microseconds in this test"}
