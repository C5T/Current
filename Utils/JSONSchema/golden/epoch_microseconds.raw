{"Object":{"fields":{"us":{"Integer":{"sum":2552763600000000,"sum_squares":2.63698200971568e30,"can_be_unsigned":true,"can_be_microseconds":true,"instances":3,"nulls":1}},"comment":{"String":{"values":{"expecting std::chrono::microseconds in this test":1,"date -d 'Aug 19 2013' +'%s000000'":1,"date -d 'Aug 19 1993' +'%s000000'":1,"date -d 'Aug 19 1983' +'%s000000'":1},"counters":[[1,"date -d 'Aug 19 1983' +'%s000000'"],[1,"date -d 'Aug 19 1993' +'%s000000'"],[1,"date -d 'Aug 19 2013' +'%s000000'"],[1,"expecting std::chrono::microseconds in this test"]],"instances":4,"nulls":0}}},"instances":4,"nulls":0}}