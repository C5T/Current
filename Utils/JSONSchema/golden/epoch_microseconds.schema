// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Object) {
  CURRENT_FIELD(us, std::chrono::microseconds);
  CURRENT_FIELD(comment, std::string);
};

using Schema = Schema_Object;
