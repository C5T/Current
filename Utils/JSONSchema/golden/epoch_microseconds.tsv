Field	Type	Set	Unset/Null	Values	Details
Schema	Object	4	0	2 fields	us, comment
Schema.us	Integer	3	0	Mean 8.50921e+14, StdDev 3.93608e+14
Schema.comment	String	4	0	4 distinct values, "expecting std::chrono::microseconds in this test":1, "date -d 'Aug 19 2013' +'%s000000'":1, "date -d 'Aug 19 1993' +'%s000000'":1, "date -d 'Aug 19 1983' +'%s000000'":1
